src/main.js

```diff
@@ -7,6 +7,14 @@ document.addEventListener("DOMContentLoaded", function (event) {
   let ignoreTextChange = false;
   let newNoteLoad = true,
     didToggleFullScreen = false;
+  
+  // Keep this variable up top so that it only needs to be loaded once.
```

Comment:
This comment is not necessary as it's typically clear why we would define something up top ;)